plugins/arOaiPlugin/modules/arOaiPlugin/actions/identifyComponent.class.php

```diff
@@ -35,6 +35,7 @@ public function execute($request)
     list ($this->earliestDatestamp) = Propel::getConnection()->query('SELECT MIN('.QubitObject::UPDATED_AT.') FROM '.QubitObject::TABLE_NAME)->fetch();
     $this->earliestDatestamp = date_format(date_create($this->earliestDatestamp), 'Y-m-d\TH:i:s\Z');
 
+    $this->baseUrl = QubitSetting::getByName('siteBaseUrl')->getValue(array('cultureFallback' => true));
```

Comment:
@mcantelon Are you sure you want the translated version of "siteBaseUrl" here (with fallback)?  I would expect:
`$this->baseUrl = QubitSetting::getByName('siteBaseUrl')->getValue(array('sourceCulture' => true));`

Reply:
Ah, copied that from the existing code and didn't notice it. I'll fix that.

Reply:
Fixed!